docs/documentation/features/frameworkUtilities.md

```diff
@@ -11,4 +11,67 @@ In this chapter useful services/utilities of the Eclipse SmartHome project are d
 The `NetworkAddressService` is an OSGi service that can be used like any other OSGi service by adding a service reference to it. Its OSGi service name is `org.eclipse.smarthome.network`.
 A user can configure his default network address via Paper UI under `Configuration -> System -> Network Settings`.
 One can obtain the configured address via the `getPrimaryIpv4HostAddress()` method on the service.
-This service is useful for example in the `ThingHandlerFactory` or an `AudioSink` where one needs a specific IP address of the host system to provide something like a `callback` URL.
\ No newline at end of file
+This service is useful for example in the `ThingHandlerFactory` or an `AudioSink` where one needs a specific IP address of the host system to provide something like a `callback` URL.
+
+## Caching
+
+The framework provides some caching solutions for common scenarios.
+
+### Simple expiring and reloading cache
+
+A common usage case is in a `ThingHandler` to encapsulate one value of an internal state and attach an expire time on that value. A cache action will be called to refresh the value if it is expired. This is what `ExpiringCache` implements. If `handleCommand(ChannelUID channelUID, Command command)` is called with the "RefreshType" command, you just return `cache.getValue()`. 
+
+It is a good practice to return as fast as possible from the `handleCommand(ChannelUID channelUID, Command command)` method to not block callers especially UIs.
+Use this type of cache only, if your refresh action is a quick to compute, blocking operation. If you deal with network calls, consider the next cache implementation instead.
```

Comment:
the "next" cache implementation

Perhaps you could state "the asynchronously cache implementation". Just to not depend on the order.